app/controllers/account_controller.rb

```diff
@@ -3,11 +3,11 @@
 class AccountController < ApplicationController
   before_action :authenticate_user!
 
-  before_action :set_user_details, only: [:settings, :settings_edit, :settings_update]
+  before_action :set_user_details, only: [:profile, :profile_edit, :profile_update]
 
   before_action :request_detail_params, only: [:request_details]
-  before_action :settings_edit_params, only: [:settings_edit]
-  before_action :settings_update_params, only: [:settings_update]
+  before_action :settings_edit_params, only: [:profile_edit]
+  before_action :settings_update_params, only: [:profile_update]
```

Comment:
Nit: should the actions and form names be changed as well for consistency, or does settings = profile + other stuff

Reply:
yeah, can do